src/gtc/cuir/oir_to_cuir.py

```diff
@@ -59,19 +59,33 @@ def visit_FieldAccess(
         accessed_fields: Set[str],
         **kwargs: Any,
     ) -> Union[cuir.FieldAccess, cuir.IJCacheAccess, cuir.KCacheAccess]:
+        data_index = [
+            self.visit(
+                index,
+                ij_caches=ij_caches,
+                k_caches=k_caches,
+                accessed_fields=accessed_fields,
+                **kwargs,
+            )
+            for index in node.data_index
+        ]
```

Comment:
```suggestion
        data_index = self.visit(
            node.data_index,
            ij_caches=ij_caches,
            k_caches=k_caches,
            accessed_fields=accessed_fields,
            **kwargs,
        )
```
Eve can directly handle lists, so the list comprehension should not be necessary. But feel free to keep it as is if you think it improves readability.